src/bootstrap/src/core/build_steps/doc.rs

```diff
@@ -604,6 +604,15 @@ impl Std {
     ) -> Self {
         Std { build_compiler, target, format, crates: vec![] }
     }
+
+    pub(crate) fn new(
```

Comment:
While this is tempting, I don't think this is a divergence which benefits us to carry.

Reply:
I added `with_crates(mut self, crates: Vec<String>)` instead.